manifests/m/Mozilla/Firefox/89.0.2/Mozilla.Firefox.locale.de-DE.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Mozilla.Firefox
+PackageVersion: 89.0.2
+PackageLocale: de-DE
+Publisher: Mozilla
+PublisherUrl: https://www.mozilla.org/de/
+PublisherSupportUrl: https://support.mozilla.org/de/
+PrivacyUrl: https://www.mozilla.org/de/privacy/websites
+Author: Mozilla Foundation
+PackageName: Mozilla Firefox
+PackageUrl: https://www.mozilla.org/de/firefox/
+License: Mozilla Public License Version 2.0
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0
+CopyrightUrl: https://www.mozilla.org/en-US/foundation/trademarks/policy
+ShortDescription: Mozilla Firefox
+Description: Firefox Browser
```

Comment:
Could you add a proper Description and ShortDescription?
This is actually what shows up when you do `winget show Firefox`

It can be as simple as the translation of
`Mozilla Firefox is free and open source software, built by a community of thousands from all over the world.`

and

`Firefox Browser, also known as Mozilla Firefox or simply Firefox, is a free and open-source web browser developed by the Mozilla Foundation and its subsidiary, the Mozilla Corporation. Firefox uses the Gecko layout engine to render web pages, which implements current and anticipated web standards. In 2017, Firefox began incorporating new technology under the code name Quantum to promote parallelism and a more intuitive user interface. Firefox is officially available for Windows 7 or newer, macOS, and Linux. Its unofficial ports are available for various Unix and Unix-like operating systems including FreeBSD, OpenBSD, NetBSD, illumos, and Solaris Unix.`

Reply:
I added a better description

Reply:
Thank you :)